### DISCUSSION TITLE
Does Omnibug see events firing in server side implementations GTM?

### DISCUSSION THREAD

**Discussion by Vermilyer** [MisterPhilip/omnibug#196] [Q&A] [answered]
We recently implemented a server-side google tag manager & moved most Google Analytics Events and marketing pixels to fire from the server. I originally thought Omni-bug would no longer see those since they are not firing client-side, But I'm still seeing them. I guess my question is how is that possible? How does Omni-bug see what's firing on our server? 
It has me wondering if our Server-side GTM is not configured correctly

**Comment by MisterPhilip** [comment, marked as answer]
@Vermilyer - it will only see any tags that fire on the client side. However, the intiial request to Server-Side GTM will still be visible. I haven't looked in a little while, but from what I remember SS GTM uses gtag with a Google Analytics client ID, so I believe I am capturing it as Google Analytics in Omnibug now, with "Google Analytics Host" differentiating if it is a request to server side or not.
